conformance/setup.go

```diff
@@ -354,6 +358,19 @@ func RunOnlyIfNot(v bool) {
 	}
 }
 
+func Warn(message string) {
+	fmt.Fprintf(colorable.NewColorableStderr(), "\n")
+	// print message
+	msgcolor := color.New(color.FgMagenta).FprintfFunc()
+	msgcolor(colorable.NewColorableStderr(), formatter.Fi(2, "\nWARNING: "+message))
+	fmt.Fprintf(colorable.NewColorableStderr(), "\n")
+	// print file:line
+	_, file, line, _ := runtime.Caller(1)
+	flcolor := color.New(color.FgWhite).FprintfFunc()
+	flcolor(colorable.NewColorableStderr(), formatter.Fi(2, "\n"))
+	flcolor(colorable.NewColorableStderr(), formatter.Fi(2, "%s:%d\n", file, line))
```

Comment:
Wondering if we can simply do this warning with log/fmt package as not to bring in additional go mods?

Reply:
We could. However, the colors give important visual feedback IMO.

https://github.com/onsi/ginkgo/blob/master/formatter/formatter_test.go#L51

https://github.com/onsi/ginkgo/blob/master/formatter/formatter.go#L60

^ let me try this instead?

Reply:
------------------------------
••
    WARNING: image manifest with no layers is not supported

    /local/rchincha/go/src/github.com/opencontainers/distribution-spec/conformance/02_push_test.go:366                                                                    ••••••••
------------------------------

^ with the right colors still happens.